agent/rpm/pbench-agent.spec.j2

```diff
@@ -95,26 +95,35 @@ fi
 %posttrans
 
 %files
-%defattr(775,pbench,pbench,775)
+%defattr(664,root,root,775)
+/%{installdir}/ansible
 /%{installdir}/config
-
 /%{installdir}/lib
-/%{installdir}/ansible
-/%{installdir}/base
 
+/%{installdir}/base
 /%{installdir}/VERSION
 /%{installdir}/SEQNO
 /%{installdir}/SHA1
 /%{installdir}/profile
 
-%ghost %attr(0400,pbench,pbench) /%{installdir}/id_rsa
+%config(noreplace) %attr(0664,root,root) /%{installdir}/config/pbench-agent.cfg
 
-%defattr(775,pbench,pbench,775)
+%defattr(775,root,root,775)
 /%{installdir}/util-scripts
+%attr(664,root,root) /%{installdir}/util-scripts/README
 /%{installdir}/tool-scripts
+%attr(664,root,root) /%{installdir}/tool-scripts/perf.README
+%attr(664,root,root) /%{installdir}/tool-scripts/README
 /%{installdir}/bench-scripts
+%attr(664,root,root) /%{installdir}/bench-scripts/pbench-fio.md
+%attr(664,root,root) /%{installdir}/bench-scripts/pbench-uperf.md
+%attr(664,root,root) /%{installdir}/bench-scripts/postprocess/BenchPostprocess.pm
+%attr(664,root,root) /%{installdir}/bench-scripts/templates/fio.job
+%attr(664,root,root) /%{installdir}/bench-scripts/templates/fio-shared-fs.job
 
 # stockpile
-%defattr(775,pbench,pbench,775)
+%defattr(664,root,root,775)
 /%{installdir}/stockpile
-
+%attr(775,root,root) /%{installdir}/stockpile/import_roles.sh
+%attr(775,root,root) /%{installdir}/stockpile/roles/openshift-cluster-topology/files/openshift_config_scraper.py
+%attr(775,root,root) /%{installdir}/stockpile/roles/openstack_common/files/openstack-config-parser.py
```

Comment:
Ditto the previous comment, except swap the permissions masks.  😀

Reply:
Apparently, this works the other way around, with only the executable files overridden from the directory default.

I still question the `ww-` here, since I don't think we want hardly any of our files (and certainly none of our scripts) to be modified, and I'm not crazy about allowing modifying directories by default either.

Reply:
Yeah, I'm not a fan of granting write-access where it's not needed, but, since we're granting it only to `root` and the guys in his group, I'm telling myself that it's not worth quibbling over.

Reply:
As above, these permissions are historical.

I've always wanted to change them.

So ... done.

Reply:
See also #2407.